kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -537,8 +524,9 @@ public Subscription create(Subscription subscription) {
     }
 
     public List<Subscription> findPendingSubscriptionsFromDate(DateTime startDate, int page, int pageSize) {
-        return subscriptionDataService.findByStatusAndStartDate(SubscriptionStatus.PENDING_ACTIVATION, startDate,
-                new QueryParams(page, pageSize));
+        return subscriptionDataService.findByStatusAndStartDate(SubscriptionStatus.PENDING_ACTIVATION, startDate, new QueryParams(page, pageSize));
     }
 
+    @CacheEvict(value = {"pack-type", "pack-name", "pack-all" }, allEntries = true)
```

Comment:
Yes, in serverEventRelay instead of calling sendEventMessage
call broadcastEventMessage although I'm not sure if your subscribe to that
the same way you do to a topic message.

On Mon, Sep 21, 2015 at 5:34 PM, Frank Huster <notifications@github.com>
wrote:

> In
> kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java
> <https://github.com/motech-implementations/mim/pull/590#discussion_r40044023>
> :
>
> >      }
> >
> > +    @CacheEvict(value = {"pack-type", "pack-name", "pack-all" }, allEntries = true)
>
> Yeah I just thought about that in the car on the drive back.
> Don't we have a way for all nodes to subscribe to a special broadcast-like
> motech event?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/590/files#r40044023>.
>